tests/jsTests/circleOfTrustsSpec.js

```diff
@@ -0,0 +1,214 @@
+/**
+ * Suite : Tests for success response modal
+ * Description : This is a test suite for checking for a modal (a prompt)
+ * when the response is a success
+ */
+describe("Tests for success response modal", function() {
+	var msg = 'This is a test message when the response is a success';
+	beforeEach(function() {
+		jasmine.clock().install();
+		setFixtures('<div class="element"></div>\
+		<div class="processing"></div>\
+		<div class="ui modal">\
+			<div class="header"></div>\
+			<div class="content"></div>\
+			<div class="actions">\
+				<div class="ui cancel green button">Close</div>\
+			</div>\
+		</div>');
+		$('.ui.modal').hide();
+
+		// mock modal show call
+		spyOn($.fn, "modal").and.returnValue($('.ui.modal').show());
+		showResponse($('element'), { response: true, message: msg });
+		// creating delay of 400ms
+		jasmine.clock().tick(4000);
+	});
+
+	afterEach(function() {
+		jasmine.clock().uninstall();
+	});
+
+	it("should be visible", function() {
+		expect($('.content')).toContainText(msg);
+		expect($('.ui.modal')).toBeVisible();
+	});
+});
+
+/**
+ * Suite : Tests for failed response modal
+ * Description : This is a test suite for a failed response modal
+ * in case of a failure
+ */
+describe("Tests for failed response modal", function() {
```

Comment:
Done here https://github.com/systers/FirstAide-web/pull/264/commits/a1520b54287ea7c45e785b2100053401cb49cb4a Completely rewritten the function and added the tests.